### PR TITLE
Maintain current functionality with gh repo create rewrite

### PR DESCRIPTION
**Pull request by ArnaudLier**
Flag --confirm / -y has been deprecated, passing any argument skips confirmation prompt.

### CODE REVIEW

**Comment by driesvints** [comment]
Where is the news about this? I can't find anything in the GitHub CLI release notes.

**Comment by ArnaudLier** [comment]
I didn't check the release notes but here is the pull requests that deprecates the flags: https://github.com/cli/cli/pull/4578
Maybe we should add the --source flag to maintain current functionality (the `origin` remote isn't added by default anymore).

**Review by driesvints — APPROVED** [review]

**Comment by driesvints** [comment]
Thanks @ArnaudLier

File: src/NewCommand.php [hunk 1]
```diff
@@ -288,7 +288,7 @@ protected function pushToGitHub(string $name, string $directory, InputInterface
         $branch = $input->getOption('branch') ?: $this->defaultBranch();
 
         $commands = [
-            "gh repo create {$name} -y {$flags}",
+            "gh repo create {$name} --source=. {$flags}",
```
Comment (driesvints) [inline src/NewCommand.php:291]: Why did you added this?

Comment (ArnaudLier) [reply src/NewCommand.php:291]: To maintain current functionality, the installer can't push the git log if there is no origin remote:
> For example, running gh repo create myrepo --public in a local repository will no longer add a remote. It assumes you just want to create a new repository on GitHub. To create a remote repo from your local git repo, use gh repo create myrepo --public --source=. instead

**Pull request merged**